stackslib/src/chainstate/stacks/auth.rs

```diff
@@ -1102,112 +1414,15 @@ mod test {
             // hash mode
             SinglesigHashMode::P2PKH as u8,
             // signer
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
-            0x11,
+            0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11, 0x11,
```

Comment:
Thanks for cleaning up this unhelpful artifact of `rustfmt`.  Makes the test a lot more legible :+1: